python/lsst/sims/catUtils/mixins/sncat.py

```diff
@@ -110,33 +125,61 @@ def observedIndices(self):
         bandPassNames = self.obs_metadata.bandpass
         return [self.lsstBandpassDict.keys().index(x) for x in bandPassNames]
 
-    def get_snid(self):
-        # Not necessarily unique if the same galaxy hosts two SN
-        # Use refIdCol to access the relevant id column of the dbobj
-        # Should revert to galTileID for galaxyTiled catalogDBObj and
-        # id for galaxyObj catalogDBObj
-        # (email from Scott)
-        return self.column_by_name(self.refIdCol)
+    @compound('TsedFilepath', 'TmagNorm')
+    def get_phosimVars(self):
```

Comment:
I changed everything back TsedFilepath to get it working. If you want I can try a different branch?